justfile

```diff
@@ -50,6 +50,7 @@
 
 @install:
   uv sync --frozen --all-extras
+  uv pip install truststore || true # truststore is not available for Python 3.9
```

Comment:
Let's just add this as a dev requirement with a marker for python >= 3.10.

Reply:
unfortimety that makes  requirements unsatifiable can i just do verify false instead of truststore ? @07pepa  that would semplify thing

Reply:
Interesting. The error message makes me think something isn't resolving correctly. I'm going to mess around with it some in a little while to see if I can get it working. For now go ahead with installing it separately.

Reply:
I just pushed a change that solved this issue for me. Let me know if it is working on your end also.

Reply:
works